deck-functional-tests/docker/publish-ci.sh

```diff
@@ -0,0 +1 @@
+gcloud docker -- push gcr.io/hebridean-sun/deck-functional-tests:ci
```

Comment:
Do we need to tie these to GCP?
That means OSS users on other platforms cant fully benefit.
At least structure this to provide guidance for other platform providers to accomodate themselves and let them do the work. But provide the direction.

Also should there be a #!/usr/bin/bash env here?